### ISSUE TITLE
Adding modifiers to overlay content breaks querying

### ISSUE THREAD

**Issue by nrivard** [nalexn/ViewInspector#11]
Consider this `View.body` function:
```swift
var body: some View {
    EmptyView()
        .overlay(Text("overlaid").offset())
}
```

and the following test:
```swift
    func testModifiedOverlayInspection() throws {
        let text = "Abc"
        let sut = try EmptyView().overlay(Text(text).offset())
            .inspect().emptyView().overlay().text().string()
        XCTAssertEqual(sut, text)
    }
```

This fails (`/<unknown>: failed: caught error: Type mismatch: ModifiedContent<Text, _OffsetEffect> is not Text`).

So I tried to ask for `offset()` first but that also fails (`/<unknown>: failed: caught error: ModifiedContent<Text, _OffsetEffect> does not have 'offset' modifier`)

**Comment by nalexn** [comment]
Nice catch! Fixed in `v0.3.6`

**Comment by nrivard** [comment]
Indeed fixed, thank you! Feel free to close this issue.
